README.md

```diff
@@ -101,6 +108,67 @@ SPRING_PROFILES_ACTIVE=local ./gradlew bootRun
 
 Then visit [http://localhost:8080/health](http://localhost:8081/health).
 
+### Checking the app has started successfully:
+If using docker, your app is probably exposed at `localhost:8080`.  
+Call http://localhost:8080/health with a browser to get app health info.
+
+
+## Querying Athena
+### Overview
+The process by which the API queries Athena is:
+1. API acquires a role from the Cloud Platform cluster it is deployed in (e.g. dev is deployed in [hmpps-electronic-monitoring-datastore-dev](https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-electronic-monitoring-datastore-dev))
+2. The [AthenaClient](src/main/kotlin/uk/gov/justice/digital/hmpps/electronicmonitoringdatastoreapi/client/AthenaClient.kt) requests a `CredentialsProvider` from the [AthenaAssumeRoleService](src/main/kotlin/uk/gov/justice/digital/hmpps/electronicmonitoringdatastoreapi/client/AthenaAssumeRoleService.kt)
```

Comment:
Has this been renamed to the EMDatastoreClient?

Reply:
Should the `AthenaAssumeRoleService` also be renamed to the `EMDatastoreAssumeRoleService` for consistency?